tests/text.php

```diff
@@ -27,7 +27,7 @@
  * @link      http://pear.php.net/package/Image_Canvas
  */
 
-require_once 'Image/Canvas.php';
+require_once 'vendor/autoload.php';
```

Comment:
Won't let this stop a merge now, but I'd prefer a more flexible approach here that doesn't force a composer installation before someone could run the testsuite.  Just a simple existence check on the vendor autoload in order to require it, and fall back to the old PEAR-style require otherwise.

Reply:
Good remark but I'm not sure I understand all the details of it. Just testing the existence of `vendor/autoload.php` would be enough ? Do you know a package that does that (so I can take inspiration from it) ?

Reply:
```
if (file_exists('vendor/autoload.php')) {
    // use composer if available
    require_once 'vendor/autoload.php';
} else {
    // otherwise rely on classic PEAR include_path
    require_once 'Image/Canvas.php';
}
```